rust-runtime/aws-smithy-experimental/src/hyper_1_0.rs

```diff
@@ -623,6 +614,10 @@ where
         let _ = (self.tcp_connector_fn)();
         Ok(())
     }
+
+    fn connector_metadata(&self) -> Option<ConnectorMetadata> {
+        Some(ConnectorMetadata::new("hyper-1", None))
```

Comment:
**correctness**: Wouldn't this be `ConnectorMetadata::new("hyper", "1") or `"1.x"`

Reply:
I'll do "1.x"